hw/ip/dcd/doc/checklist.md

```diff
@@ -115,8 +115,8 @@ Review        | Signoff date            | Not Started |
 
  Type         | Item                                  | Resolution  | Note/Collaterals
 --------------|---------------------------------------|-------------|------------------
-Documentation | [DV_DOC_DRAFT_COMPLETED][]            | Not Started | [DCD DV Plan]({{<relref "hw/ip/dcd/doc/dv_plan" >}})
-Documentation | [DV_PLAN_COMPLETED][]                 | Not Started | [DCD Testplan]({{<relref "hw/ip/dcd/doc/dv_plan/index.md#testplan" >}})
+Documentation | [DV_DOC_DRAFT_COMPLETED][]            | Not Started | [DCD DV Plan]()
```

Comment:
The DV doc exists. This needs to change to:
```
Documentation | [DV_DOC_DRAFT_COMPLETED][]            | Not Started | [DCD DV document]({{<relref "hw/ip/dcd/doc/dv" >}})
```

I think Hugo might complain though - so you would need to add `hw/ip/dcd/doc/_index.md` (maybe just put the title in there until Eric can update it). You can then revert the other changes in this commit.

Reply:
sounds good.